content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,179 @@
+  ([Issue #30683](https://github.com/istio/istio/issues/30683))
+
+- **Fixed** a bug where the EnvoyFilter `HTTP_FILTER` didn't support `INSERT_FIRST`.
+  ([Issue #31573](https://github.com/istio/istio/issues/31573))
+
+- **Fixed** an issue where services with `PASSTHROUGH` load balancing were always sent mTLS traffic, even if the destinations did not support mTLS.
+  ([Issue #23494](https://github.com/istio/istio/issues/23494))
+
+- **Fixed** a bug where `EnvoyFilter` with service match did not work for inbound clusters.
+
+## Security
+
+- **Added** an experimental feature to allow dry-run of an `AuthorizationPolicy` without actually enforcing the policy.
+ ([Usage](/docs/tasks/security/authorization/authz-dry-run/), [Design](https://docs.google.com/document/d/1xQdZsEgJ3Ld2qebfT3EJkg2COTtCR1TqBVojmnvI78g), [PR](https://github.com/istio/api/pull/1933))
+
+- **Updated** configuration to sign istiod certificates using Kubernetes CA (`PILOT_CERT_PROVIDER=kubernetes`) will not be honored in
+clusters with version 1.22 and greater.
+  ([Issue #22161](https://github.com/istio/istio/issues/22161))
+
+- Improved the experimental [External Authorization](/docs/tasks/security/authorization/authz-custom/) feature with new capabilities:
+    - **Added** the `timeout` field to configure the timeout (default is `10m`) between the `ext_authz` filter and the external service.
+    - **Added** the `include_additional_headers_in_check` field to send additional headers to the external service.
+    - **Added** the `include_request_body_in_check` field to send the body to the external service.
+    - **Supported** prefix and suffix match in the `include_request_headers_in_check`, `headers_to_upstream_on_allow` and `headers_to_downstream_on_deny` field.
+    - **Deprecated** the `include_headers_in_check` field with the new `include_request_headers_in_check` field for better naming. ([Reference](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-EnvoyExternalAuthorizationHttpProvider), [PR](https://github.com/istio/api/pull/1926))
+
+- **Added** experimental option to configure Envoy to fetch the JWKS by itself. This should be enabled if the `jwks_uri` is a mesh cluster URL for mTLS and has other benefits like retries, JWKS caching etc.
+This is disabled by default and can be enabled by setting `PILOT_JWT_ENABLE_REMOTE_JWKS` to true.
+  ([Issue #28742](https://github.com/istio/istio/issues/28742))
+
+- **Added** an environment variable `PILOT_JWT_PUB_KEY_REFRESH_INTERVAL` to configure the interval of
+istiod fetching the `jwks_uri` for the JWKS public key. Users can set the refresh interval with
+`--set values.pilot.env.PILOT_JWT_PUB_KEY_REFRESH_INTERVAL=<duration>` during installation.
+The default interval is `20m`. Valid time units are "ns", "us", "ms", "s", "m", "h".
+
+- **Added**  istiod JWT public key refresh job will now retry the failed fetch of the `jwks_uri` with exponential backoff.
```

Comment:
Extra space after Added can be removes.  Is the job added, or updated? Sounds like this should read Update the istiod JWT public key refresh job to retry......